packages/nestjs-user/src/user.types.ts

```diff
@@ -1,4 +1,11 @@
+import { PasswordStrengthTransformOptionsInterface } from '@concepta/nestjs-common';
+import { PasswordStrengthEnum } from '@concepta/nestjs-password';
+
 export enum UserResource {
   'One' = 'user',
   'Many' = 'user-list',
 }
+
+export type PasswordStrengthTransform = (
+  options: PasswordStrengthTransformOptionsInterface,
+) => PasswordStrengthEnum | null;
```

Comment:
undefined is better than null
```suggestion
) => PasswordStrengthEnum | undefined;
```